rendering_styles/default.render.xml

```diff
@@ -5413,7 +5413,14 @@
 					</switch>
 					<case minzoom="13" textSize="12" tag="natural" value="gorge" textOnPath="true" textOrder="44" textDy="0"/>
 					<case minzoom="13" textSize="12" tag="natural" value="couloir" textOnPath="true" textOrder="44" textDy="0"/>
-					<case minzoom="10" textSize="13" tag="region:type" value="mountain_area" textOrder="43"/>
+					<switch tag="region:type" value="mountain_area" textHaloRadius="5" textOrder="1" textDy="0">
```

Comment:
textOrder="1" here will hide many important features like city and state names. Default style is general purpose style and mountain areas are not a feature most users want to see instead of city names.

Reply:
I suggest 33 here (over cities with population 70000 but under cities without population tag (or population>200000))

Reply:
OK, thanks for your suggestion. I'll do again all tests with *textOrder="33"*